packages/grafana-i18n/src/internal/index.ts

```diff
@@ -24,3 +24,5 @@ export {
   getResolvedLanguage,
   initTranslations,
```

Comment:
Should we stop exporting `initTranslations`

Reply:
i this just an idle observation unrelated to this pr? Just making sure i follow properly

Reply:
@joshhunt sorry I was OOO yesterday but as we aren't using the `initTranslations` function from grafana core we should make sure to remove it from the exports, right? It's now an internal function of `initializeI18n` function.